### ISSUE TITLE
when i enter this command " minikube service app1" it shows below error

### ISSUE THREAD

**Issue by KrishnappaL** [kubernetes/minikube#19070]
### What Happened?

 Exiting due to SVC_UNREACHABLE: service not available: no running pod for service app1 found

### Attach the log file

[log.txt](https://github.com/user-attachments/files/15830427/log.txt)

### Operating System

None

### Driver

None

**Comment by BerkAkipek** [comment]
Hello my friend make sure you write right selector in your service file minikube can't detect your running pod.

**Comment by Ritikaa96** [comment]
/kind support
Hey @KrishnappaL does the problem still exist after checking the selector and exposing port?

**Comment by spowelljr** [comment]
Hi @KrishnappaL, could you run `kubectl get pods -A` and upload the output, also include your YAML file of your service so we could debug this, thank you.

**Comment by KrishnappaL** [comment]
Yes,that's correct.Kubernetes not detect a service pod, because after I run docker image as a pod I deleted that pod.I only done this for understanding the working of kubernets.

**Comment by spowelljr** [comment]
Thanks for the response @KrishnappaL, since it seems that this behavior was expected is it okay if we close this issue?

**Comment by KrishnappaL** [comment]
Okay
